doc/source/MANUAL.rst

```diff
@@ -73,6 +73,10 @@ A full example config file is::
 The ``group_regex`` option is used to specify is used to provide a scheduler
 hint for how tests should be divided between test runners. See the
 :ref:`group_regex` section for more information on how this works.
+You can also specify the ``parallel_class=True`` instead of
```

Comment:
Can we make this a separate paragraph (just add a blank line). Also I think we should reorder it so we say that it groups by class before saying the equivalent regex. Basically something like:

```suggestion

You can also specify the ``parallel_class=True`` instead of group_regex to group tests in the stestr scheduler together by class. Since this is a common use case this enables that without needing to memorize the complicated regex for ``group_regex`` to do this.
```

Reply:
Thanks for the suggestion!